src/main/java/net/minecraftforge/client/event/render/EntityViewRenderEvent.java

```diff
@@ -0,0 +1,390 @@
+        public float getGreen()
+        {
+            return green;
+        }
+
+        /**
+         * Sets the new green color value of the fog.
+         *
+         * @param green the new blue color value
+         */
+        public void setGreen(float green)
+        {
+            this.green = green;
+        }
+
+        /**
+         * {@return the blue color value of the fog}
+         */
+        public float getBlue()
+        {
+            return blue;
+        }
+
+        /**
+         * Sets the new blue color value of the fog.
+         *
+         * @param blue the new blue color value
+         */
+        public void setBlue(float blue)
+        {
+            this.blue = blue;
+        }
+    }
+
+    /**
+     * Fired to allow altering the angles of the player's camera.
+     * This can be used to alter the player's view for different effects, such as applying roll.
+     *
+     * <p>This event is not {@linkplain Cancelable cancelable}, and does not {@linkplain HasResult have a result}. </p>
+     *
+     * <p>This event is fired on the {@linkplain MinecraftForge#EVENT_BUS main Forge event bus},
+     * only on the {@linkplain LogicalSide#CLIENT logical client}. </p>
+     *
+     * @see ForgeHooksClient#onCameraSetup(GameRenderer, Camera, float)
+     */
+    public static class CameraSetup extends EntityViewRenderEvent
+    {
+        private float yaw;
+        private float pitch;
+        private float roll;
+
+        public CameraSetup(GameRenderer gameRenderer, Camera camera, double partialTick, float yaw, float pitch, float roll)
+        {
+            super(gameRenderer, camera, partialTick);
+            this.setYaw(yaw);
+            this.setPitch(pitch);
+            this.setRoll(roll);
+        }
+
+        /**
+         * {@return the yaw of the player's camera}
+         */
+        public float getYaw()
+        {
+            return yaw;
+        }
+
+        /**
+         * Sets the yaw of the player's camera.
+         *
+         * @param yaw the new yaw
+         */
+        public void setYaw(float yaw)
+        {
+            this.yaw = yaw;
+        }
+
+        /**
+         * {@return the pitch of the player's camera}
+         */
+        public float getPitch()
+        {
+            return pitch;
+        }
+
+        /**
+         * Sets the pitch of the player's camera.
+         *
+         * @param pitch the new pitch
+         */
+        public void setPitch(float pitch)
+        {
+            this.pitch = pitch;
+        }
+
+        /**
+         * {@return the roll of the player's camera}
+         */
+        public float getRoll()
+        {
+            return roll;
+        }
+
+        /**
+         * Sets the roll of the player's camera.
+         *
+         * @param roll the new roll
+         */
+        public void setRoll(float roll)
+        {
+            this.roll = roll;
+        }
+    }
+
+    /**
+     * Fired for altering the raw field of vision (FOV).
```

Comment:
Shouldn't this be 'field of view'? The event is also called FieldOfView and 'field of vision' sounds a bit odd, although that might just be me.

Reply:
I think I might have mixed the word `vision` and `view` there. It should be `field of view`.